packages/fxa-auth-server/lib/payments/stripe.js

```diff
@@ -73,6 +73,10 @@ async function cachedResult(log, redis, cacheKey, cacheTtl, refreshFunction) {
   return result;
 }
 
+async function deleteCachedResult(log, redis, cacheKey) {
+  await redis.del(cacheKey);
+}
```

Comment:
The function doesn't do anything with `log`.

Reply:
Wrapped the delete in a try/catch that logs the error